<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mobileLogin</class>
 <widget class="QWidget" name="mobileLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="WCommonEdit" name="verificationCodeEdit" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>0</y>
     <width>369</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WVerificationCode" name="verificationCodePic" native="true">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>0</y>
     <width>159</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WCommonEdit" name="mobileNumEdit" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>80</y>
     <width>425</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WButton" name="getCodeBtn" native="true">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>80</y>
     <width>103</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="estimatedLable">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>152</y>
     <width>249</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estimated time of code in 43 seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="resendLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>152</y>
     <width>88</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Resend Code</string>
   </property>
  </widget>
  <widget class="WCommonEdit" name="codeEdit" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>193</y>
     <width>544</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WCheckBox" name="loggedCheckBox" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>258</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 20px; height: 20px;}

QCheckBox::indicator:unchecked {background-image: url(:/QtTest/icon/check_sel.png);}


QCheckBox::indicator:checked {background-image: url(:/QtTest/icon/chebox_select.png);}</string>
   </property>
   <property name="text">
    <string>Keep me logged in</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WCommonEdit</class>
   <extends>QWidget</extends>
   <header location="global">WCommonEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WButton</class>
   <extends>QWidget</extends>
   <header location="global">wbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WCheckBox</class>
   <extends>QWidget</extends>
   <header location="global">wcheckbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WVerificationCode</class>
   <extends>QWidget</extends>
   <header location="global">WVerificationCode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
